### ISSUE TITLE
Incorrect display of objects behind VDB

### ISSUE THREAD

**Issue by reflectline** [eidosmontreal/unreal-vdb#91]
Hello, I have encountered this problem, for some reason objects that are behind the cloud are projected through it, please could you explain how to fix it? /UE 5.2.0/Plugin v.0.6.9 
![114042375](https://github.com/eidosmontreal/unreal-vdb/assets/118726953/1dbaeace-89d9-4c3e-80ab-5c39bca2f13d)

**Comment by Andrej730** [comment]
@HI4AN you're using principled actor / material actor or BP_VdbToVolumetricClouds ?

In case you're using principled or material actor there is no solution - 
https://github.com/eidosmontreal/unreal-vdb/blob/main/HELPME.md#i-have-sorting-issues-with-transparent-materials-andor-hair-strands-is-there-any-way-to-fix-these-

For BP_VdbToVolumetricClouds there is a solution - https://github.com/eidosmontreal/unreal-vdb/issues/48#issuecomment-1345471648

**Comment by thilamb** [comment]
Thanks @Andrej730 . 

Yes it's not very clear what your systems you're using and mixing but the answer above should cover it.

Let us know, thanks.

**Comment by reflectline** [comment]
@Andrej730. @thilamb 
This is BP from the "easy fog" set, it can be found on the marketplace. This bp is a plane with transparent animated material. I was able to solve this problem by changing the properties of its root material, changed the material domain to decal, before that there was a "surface" position. As an actor I used vdb material actor.  Sorry if my question was stupid, I don't know unreal that well, anyway thanks for your answers
![43](https://github.com/eidosmontreal/unreal-vdb/assets/118726953/cf9802f0-61df-4f79-943c-39a4b702df20)
